pkg/cli/queue/operate.go

```diff
@@ -69,7 +69,7 @@ func OperateQueue() error {
 	}
 
 	if len(operateQueueFlags.Name) == 0 {
-		return fmt.Errorf("Queue name must be specified")
+		return fmt.Errorf("queue name must be specified")
```

Comment:
This will cause CI error, could you fix it?
https://github.com/volcano-sh/volcano/pull/1552/checks?check_run_id=2883616120

Reply:
Thanks, I fixed it.